CHANGES.md

```diff
@@ -9,6 +9,7 @@
 - [THRIFT-5527](https://issues.apache.org/jira/browse/THRIFT-5527) - Compiler generated Process function will swallow exceptions defined in thrift IDL
 - [THRIFT-5605](https://issues.apache.org/jira/browse/THRIFT-5605) - Provide `ExtractIDLExceptionClientMiddleware` and `ExtractExceptionFromResult` to help client middlewares to gain access to exceptions defined in thrift IDL
 - [THRIFT-5609](https://issues.apache.org/jira/browse/THRIFT-5609) - `TJSONProtocol` is now safe to be used in `TDeserializePool`
+- [THRIFT-5423](https://issues.apache.org/jira/browse/THRIFT-5423) - Specify thrift parameter validation proposal and provide parameter validator for go language.
```

Comment:
this is actually no longer going to make the 0.17.0 release and will be in 0.18.0 instead. @Jens-G is the recommendation to create the 0.18.0 section or just remove this line?

Reply:
Usually not necessary, since the changes.md file will be [assembled during release process](https://github.com/apache/thrift/blob/master/doc/ReleaseManagement.md#release-candidate). 
But what you might want to include beforehand is [breaking changes](https://github.com/apache/thrift/blob/master/CHANGES.md#breaking-changes).

Reply:
I have just removed this line since there are no breaking changes from the user side. Or should I create a 0.18.0 section and add this line to breaking changes